### DISCUSSION TITLE
Coloring Cmdlets according to the Verb.

### DISCUSSION THREAD

**Discussion by AndreBengaly** [PowerShell/PowerShell#16613] [Enhancements]
### Summary of the new feature / enhancement

Example:

verb get/find - Green
**Get**-help

verb add - Yellow
**Add**-localuser

verb write/remove/delete - Red
**Remove**-adcomputer

**Comment by MartinGC94** [comment]
I would personally find that distracting, but if this were to be implemented it should probably be in PSReadline and whichever editor you use for PowerShell. I don't think the Parser should differentiate command tokens with and without a verb.

**Comment by iSazonov** [comment]
Ideally PowerShell should support generic formatting with flexible coloring by schemas. It's not like that now.
